sdk/cpp/core/docsgen/api/ydk/providers/netconf_provider.rst

```diff
@@ -39,13 +39,55 @@ NetconfServiceProvider
 
         Constructs an instance of ``NetconfServiceProvider`` using the provided :cpp:class:`repository<path::Repository>`
 
-        :param repository: Reference to an instance of :cpp:class:`Repository<ydk::path::Repository>`
+        :param repo: Reference to an instance of :cpp:class:`Repository<ydk::path::Repository>`
         :param address: IP address of the device supporting a netconf interface
         :param username: Username to log in to the device
         :param password: Password to log in to the device
         :param port: Device port used to access the netconf interface. Default value is 830
         :param timeout: The timeout in microseconds, -1 for infinite timeout, 0 for non-blocking
 
+    .. cpp:function:: NetconfServiceProvider(\
+        const path::Repository& repo, \
+        std::string address, \
+        std::string username, \
+        std::string private_key_path, \
+        std::string public_key_path, \
+        int port = 830, \
+        bool on_demand = true, \
+        int timeout = -1)
+
+        Constructs an instance of ``NetconfServiceProvider`` connect to a server which **has** to support model download
+
+        :param repo: Reference to an instance of :cpp:class:`Repository<ydk::path::Repository>`
+        :param address: IP address of the device supporting a netconf interface.
+        :param username: Username to log in to the device.
+        :param private_key_path: Path to private key file.
```

Comment:
Can you indicate if this is the absolute path or relative path?

Reply:
I've tried both and it takes either, but yeah I can specify that.